Water-Physics/Assets/Code/Scripts/Environment/WaterFloat.cs

```diff
@@ -6,11 +6,11 @@ namespace Code.Scripts.Environment
     public class WaterFloat : MonoBehaviour
     {
         //public properties
-        public float airDrag = 1;
-        public float waterDrag = 10;
-        public bool affectDirection = true;
-        public bool attachToSurface;
-        public Transform[] floatPoints;
+        [SerializeField] private float airDrag = 1;
```

Comment:
Begin variabelen met hoofdletter

Reply:
oké nice dat we zo veel tegen de naming conventions van C# in gaan dan

Reply:
We hebben net nog even discussie gehad. Fields (Dus zonder getter/setter. public/private maakt niet uit) gaan met camelCase. Als je een property maakt (dus met getter/setter) dan doe je PascalCase.

Reply:
Oof net alles aangepast

Reply:
oof